tailwind/ui/__stories__/ButtonLink.stories.tsx

```diff
@@ -0,0 +1,19 @@
+import type { Meta, StoryObj } from "@storybook/react/*"
+
+import { ButtonLink as ButtonLinkComponent } from "../buttons/Button"
+
+const meta = {
+  title: "Atoms / Form / ShadCn Buttons",
+  component: ButtonLinkComponent,
+} satisfies Meta<typeof ButtonLinkComponent>
+
+export default meta
+
+export const ButtonLink: StoryObj<typeof meta> = {
+  args: {
+    linkProps: {
```

Comment:
Hey not sure about this API for the ButtonLink. I think we should flip the props. We should prioritize the link props over the buttons.

```
// this
<ButtonLink href="..." buttonProps={...} />

// rather than
<ButtonLink linkProps={{ href: "..." }} />
```

That would also align with the previous implementation.

Reply:
Agree. Will flip it!